docs/core/tools/dotnet-nuget-trust.md

```diff
@@ -0,0 +1,319 @@
+---
+title: dotnet nuget trust command
+description: The dotnet nuget trust command gets or sets trusted signers to the NuGet configuration.
+author: erdembayar
+ms.date: 06/02/2021
+---
+# dotnet nuget trust
+
+**This article applies to:** ✔️ .NET 5.0.300 SDK and later versions
+
+## Name
+
+`dotnet nuget trust` - Gets or sets trusted signers to the NuGet configuration.
+
+## Synopsis
+
+```dotnetcli
+dotnet nuget trust [command] [OPTIONS]
+
+dotnet nuget trust -h|--help
+```
+
+## Description
+
+The `dotnet nuget trust` manage the trusted signers. By default, NuGet accepts all authors and repositories. These commands allow you to specify only a specific subset of signers whose signatures will be accepted, while rejecting all others. For additional usage, see [Common NuGet configurations](../../../nuget/consume-packages/configuring-nuget-behavior.md). For details on how the nuget.config schema looks like, refer to the [NuGet config file reference](../../../nuget/reference/nuget-config-file.md).
+
```

Comment:
@tdykstra 
Do I need to put relative path or absolute path for link?

Reply:
Since it's not  in the same repo it will have to be a relative URL rather than a path to the .md file:

/nuget/consume-packages/configuring-nuget-behavior

/nuget/reference/nuget-config-file

Reply:
@tdykstra 
Ok. I corrected it. Could you be able to review now?